docs/fieldnorm/software_life_cycle_processes.md

```diff
@@ -0,0 +1,104 @@
+<style type="text/css">
+    ol { list-style-type: lower-alpha; }
+</style>
+
+# Software Development Life Cycle processes
+One of the main responsibilities of bioinformaticians within a diagnostic setting involves the development and 
+maintenance of software used in diagnostic processes. 
+The IEC-62304 description of the software development life cycle (SDLC) is consistent with the practices of most 
+software-centric entities. 
+SDLC processes that are commonly used include planning, requirements, architectural design, development, testing, 
+deployment, and maintenance. Laboratories should establish documented guidelines that cover all SDLC these stages.
+
+## 1. Planning
+Planning entails developing a comprehensive software development plan. 
+This includes brainstorming the initial ideas, ideation of the final goal, determining the necessary resources, 
+creating a schedule, and establishing achievable milestones. 
+Bioinformaticians should establish a clear communication path with stakeholders during this stage to ensure 
+that the plan meets the goal. 
+Within a laboratory, software development often occurs in the background of laboratory tests. 
+The laboratory therefore needs to develop a solid strategy for change management for its tests that emphasizes all these 
+characteristics, and software development should be an integral part of this strategy. 
+Smaller-scale software development and maintenance cycles that might not affect other areas of a laboratory should 
+maintain a similar strategy, although on a smaller scale and less demanding.
+Guidelines should cover and define both large-scale and small-scale strategies.
+
+> 📝 _Note: Examples of small scale and large scale projects could be:_
+> - _Small-scale: the adaptation of a new component to existing software_
+> - _Large-scale: developing entirely new software, big refactors of existing software or changes to algorithms_
+
+
+## 2. Requirements
+This process involves gathering, documenting, and analyzing the software requirements. 
+It includes capturing both functional and non-functional requirements that the software must fulfill to meet the 
+intended use and comply with regulatory standards. 
+One example strategy for capturing this, which is also well suited for medical laboratories, is to use user stories, 
+as stakeholders in a medical laboratory have very specific needs 
+and often find it difficult to translate these into well-defined technical requirements.
+
+Guidelines should support the following processes to gather, convert, track, and update the requirements during a project:
+1. Collaboration between end users and developers when defining software requirements. 
+2. Conversion of users' requirements into small, individual development tasks.
+3. A plan for keeping in touch with end users throughout the different stages of the development process 
+   to avoid unexpected adjustments to requirements later on.
+
+
+> 📝 _Note:  
```

Comment:
The word Note is redundant here, as the text is already encased  in a box signifying a note. 

The same goes for the emoji. The text already stated that it is a note, and the box even moreso.